packages/main/src/WheelSlider.js

```diff
@@ -0,0 +1,272 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import WheelSliderTemplate from "./generated/templates/WheelSliderTemplate.lit.js";
+import Button from "./Button.js";
+import {
+	isDown,
+	isUp,
+	isTabNext,
+	isTabPrevious,
+} from "../../base/src/Keys.js";
+
+// Styles
+import WheelSliderCss from "./generated/themes/WheelSlider.css.js";
+
+/**
+ * @private
+ */
+const metadata = {
+	tag: "ui5-wheelslider",
+	properties: /** @lends sap.ui.webcomponents.main.WheelSlider.prototype */ {
+		/**
+		 * Defines whether the <code>ui5-wheelslider</code> is disabled
+		 * (default is set to <code>false</code>).
+		 * A disabled <code>ui5-wheelslider</code> can't be pressed or
+		 * focused, and it is not in the tab chain.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		disabled: {
+			type: Boolean,
+		},
+
+		/**
+		 * Defines the currently selected value
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		value: {
+			type: String,
+		},
+
+		/**
+		 * Defines the label of the wheelslider.
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		label: {
+			type: String,
+			defaultValue: "",
+		},
+
+		/**
+		 * Indicates if the wheelslider is expanded.
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
+		 */
+		_expanded: {
+			type: Boolean,
+		},
+
+		_items: {
+			type: Object,
+		},
+	},
+	slots: /** @lends sap.ui.webcomponents.main.WheelSlider.prototype */ {
+
+	},
+	events: /** @lends sap.ui.webcomponents.main.WheelSlider.prototype */ {
+		/**
+		 * Fires when the wheel slider is expanded.
+		 */
+		expand: {},
+
+		/**
+		 * Fires when the wheel slider is collapsed.
+		 */
+		collapse: {},
+
+		/**
+		 *  Fires when new value is selected.
+		 */
+		valueSelect: {
```

Comment:
Is this event fired somewhere, I could not find it, maybe it is something initially done, but then changed

Reply:
Done